tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the ekino/sonata project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\HelpersBundle\Tests\DependencyInjection;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\HelpersBundle\DependencyInjection\Configuration;
+use Symfony\Component\Config\Definition\Processor;
+
+/**
+ * @author Vincent Laurier <vincent.laurier@ekino.com>
+ */
+class ConfigurationTest extends TestCase
+{
+    public function testDefaultConfig()
+    {
+        $processor = new Processor();
+        $config    = $processor->processConfiguration(new Configuration(), []);
+
+        $this->assertEquals(self::getBundleDefaultConfig(), $config);
+    }
+
+    protected static function getBundleDefaultConfig()
```

Comment:
can you use a dataProvider instead and set the method private plz

Reply:
I replaced it by a dataProvider, but then it can only be public.